R/tspa_plot_revised.R

```diff
@@ -0,0 +1,123 @@
+tspa_plot <- function(tspa_fit,
+                      ask = FALSE,
+                      title = NULL,
+                      label_x = NULL,
+                      label_y = NULL,
+                      abbreviation = TRUE,
+                      fscore_type = NULL,
```

Comment:
If you're allowing users to specify a type, the argument should be something like `fscore_type = c("type1", "type2")`, and then use `match.arg` inside the function. See `get_fs()` for an example.

Reply:
Thanks for the suggestion. I made the revision accordingly.